src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -2045,6 +2059,273 @@ def _pp_asym_gens_output_3ph(self):
         assert "res_asymmetric_sgen_3ph" not in self.pp_output_data
         self.pp_output_data["res_asymmetric_sgen_3ph"] = pp_output_asym_gens_3ph
 
+    # pylint: disable-msg=too-many-locals
+    def _pp_update_loads(self):  # pragma: no cover
+        pp_upd_data = self.pp_update_data["controller"]["object"]
+
+        scaling = self._get_pp_attr("load", "scaling", 1.0)
+        all_load_ids = self.pp_update_data["load"].index.values
+        const_i_multiplier = self._get_pp_attr("load", "const_i_percent", 0) * scaling * (1e-2 * 1e6)
+        const_z_multiplier = self._get_pp_attr("load", "const_z_percent", 0) * scaling * (1e-2 * 1e6)
```

Comment:
```suggestion
        const_i_multiplier = self._get_pp_attr("load", "const_i_percent", 0) * scaling * 1e4
        const_z_multiplier = self._get_pp_attr("load", "const_z_percent", 0) * scaling * 1e4
```

or split the coefficients, e.g. (but please clean up a bit by reordening):

```suggestion
        percent = 1e-2
        unit_magnitude = 1e6
        scaling *= unit_magnitude
        const_i_multiplier = self._get_pp_attr("load", "const_i_percent", 0) * percent * scaling
        const_z_multiplier = self._get_pp_attr("load", "const_z_percent", 0) * percent * scaling
```